dmoj/settings.py

```diff
@@ -549,6 +549,9 @@
 PDF_STATEMENT_SAFE_EXTS = {'pdf'}
 PDF_STATEMENT_MAX_FILE_SIZE = 5242880
 
+SUBMISSION_FILE_UPLOAD_URL_PREFIX = '/submission'
```

Comment:
don't use this.
Because dmoj use the same URL to view submission.
I think we should use `/submission_file`